src/app/application.cpp

```diff
@@ -804,6 +807,30 @@ void Application::applyMemoryWorkingSetLimit()
 #endif
 }
 
+void Application::adjustThreadPriority() const
+{
+    // Workaround to raise Qt threads to be just one level higher than libtorrent threads.
+    // This is to improve responsiveness of qbt
+
+#ifdef Q_OS_WIN
+    ::SetThreadPriority(::GetCurrentThread(), THREAD_PRIORITY_ABOVE_NORMAL);
+#elif defined(Q_OS_UNIX)
+    const pthread_t currentThread = ::pthread_self();
+
+    int policy = 0;
+    sched_param param {};
+    if (::pthread_getschedparam(currentThread, &policy, &param) != 0)
+        return;
+
+    const int maxPrio = ::sched_get_priority_max(policy);
+    if (maxPrio <= -1)
+        return;
+    param.sched_priority = std::min((param.sched_priority + 1), maxPrio);
+
```

Comment:
I would remove this empty line and add it after `return;` line.

Reply:
Only Windows implementation is provided now. I couldn't find a way to do it reliably on Linux.